packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -1959,75 +1904,52 @@ Difference:
 `;
 
 exports[`.toEqual() {pass: false} expect({}).not.toEqual({}) 1`] = `
-"<dim>expect(</><red>received</><dim>).not.toEqual(</><green>expected</><dim>)</>
+"<dim>expect(</><red>received</><dim>).</>not<dim>.toEqual(</><green>expected</><dim>)</>
 
-Expected value to not equal:
-  <green>{}</>
-Received:
-  <red>{}</>"
+Expected: <green>{}</>
+Received: <red>{}</>"
 `;
 
 exports[`.toEqual() {pass: false} expect(0).toEqual(-0) 1`] = `
 "<dim>expect(</><red>received</><dim>).toEqual(</><green>expected</><dim>)</>
 
-Expected value to equal:
-  <green>-0</>
-Received:
-  <red>0</>
-
 Difference:
 
 <dim>Compared values have no visual difference.</>"
```

Comment:
This is because `jest-diff` use strict equality `===` operator when `Object.is` might be better.

That would be a separate pull request.

Reply:
we should land that on master, then rebase this, right? To ensure the message is good